src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -17,7 +17,7 @@
 public class DriverDaoImpl implements DriverDao {
     @Override
     public Driver create(Driver driver) {
-        String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
+        String query = "INSERT INTO drivers (name, licenseNumber) VALUES (?, ?)";
```

Comment:
```suggestion
        String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
```
this is the right way of giving column names, not in uppercase. If in your DB your columns are in camelCase then better change it.